### PR TITLE
Added Mill's dependency string

### PR DESCRIPTION
**Pull request by rusucosmin**

### CODE REVIEW

**Comment by julienrf** [comment]
Hey, can you add a screenshot showing the result of your changes?

**Comment by rusucosmin** [comment]
Sure, here it is:

screen shot 2018-12-15 at 11 02 30 am

**Comment by julienrf** [comment]
I think we should also show how to add a resolver, when needed, like we do here for sbt:

~~~ scala
libraryDependencies += "de.heikoseeberger" %% "akka-http-circe" % "1.21.1"
resolvers += Resolver.bintrayRepo("hseeberger", "maven")
~~~

File: model/src/main/scala/ch.epfl.scala.index.model/Release.scala [hunk 1]
```diff
@@ -130,6 +130,22 @@ case class Release(
...
+   */
+  def millInstall = {
+    def addResolver(r: Resolver) =
+      r.url map (url => s"""MavenRepository("${url}")""")
+    val artifactOperator = if (isNonStandardLib) ":" else "::"
+    List(
+      Some(
+        s"""ivy"${maven.groupId}$artifactOperator${reference.artifact}:${reference.version}""""
+      ),
+      resolver flatMap addResolver
```
Comment (julienrf) [inline model/src/main/scala/ch.epfl.scala.index.model/Release.scala]: I’m not sure we should generate the “Mill” tab at all, if the `addResolver` method returns `None`.

Comment (rusucosmin) [reply model/src/main/scala/ch.epfl.scala.index.model/Release.scala]: Why? In the [issue](**https://github.com/scalacenter/scaladex/issues/543**) @Baccata requested only the dependency. I can add an if guard in that case, but I think that it is more convenient if you can just copy paste the dependency even without a resolver.

**Comment by julienrf** [comment]
Thanks! I’m merging it. We can improve it later, if needed.

**Pull request merged**